include/wx/graphics.h

```diff
@@ -431,10 +431,13 @@ class WXDLLIMPEXP_CORE wxGraphicsContext : public wxGraphicsObject
 #if wxUSE_PRINTING_ARCHITECTURE
     static wxGraphicsContext * Create( const wxPrinterDC& dc);
 #endif
-#ifdef __WXMSW__
-#if wxUSE_ENH_METAFILE
+#if defined(__WXMSW__) && wxUSE_ENH_METAFILE
```

Comment:
Just curious, why the need to change this?

Also, the `__WXMSW__` test is probably unnecessary, if `wxUSE_ENH_METAFILE ` is always defined, it should be defined correctly, i.e. as 0 under non-MSW platforms.